src/components/common/datagrid.js

```diff
@@ -125,12 +129,19 @@ function DataGridFactory(
 
       const type = columns[columnIndex].type;
 
+      let content = null;
+      if (rowIndex) {
+        const {dataColumnMap = {}} = this.props;
+        const dataIndex = dataColumnMap[columnIndex] !== null || dataColumnMap[columnIndex] !== undefined
+          ? dataColumnMap[columnIndex] : columnIndex;
+        content = (<Cell className={`cell ${type}`} value={rows[rowIndex - 1][dataIndex]} type={type} />);
```

Comment:
`className` here will not be displayed, because you are not rendering it in `Cell`

Reply:
fixed